cmd/kubeadm/app/phases/selfhosting/selfhosting_volumes.go

```diff
@@ -32,7 +32,7 @@ import (
 type tlsKeyPair struct {
 	name string
 	cert string
-	key  string
+	key  string `datapolicy:"security-key"`
```

Comment:
this is technically a path to the key:
https://github.com/kubernetes/kubernetes/blob/3d9c6eb9e6e1759683d2c6cda726aad8a0e85604/cmd/kubeadm/app/phases/selfhosting/selfhosting_volumes.go#L235-L236

Reply:
Removed and renamed struct to `tlsKeyPairPath` to avoid confusion on next iterations.